dashboard/v1.0/.prettierrc.json

```diff
@@ -3,7 +3,8 @@
         {
             "files": [".prettierrc", ".babelrc", ".eslintrc", ".stylelintrc"],
             "options": {
-                "parser": "json"
+                "parser": "json",
+                "tabWidth": 2
```

Comment:
Please make sure that this has effect in the CI tests. They should run strictly against these rules and fail on a single out. Else, this would be pointless.